### PR TITLE
MM-47235: Fix styling of hr in status updates

### PR DESCRIPTION
**Pull request by lieut-data**
## Summary
An upstream change to CSS broke the styling for status updates:
image

Restore to the expected design:
CleanShot 2022-11-14 at 11 31 20@2x

## Ticket Link
Fixes: https://mattermost.atlassian.net/browse/MM-47235

### CODE REVIEW

File: webapp/src/components/update_post.tsx [hunk 1]
```diff
@@ -117,10 +117,11 @@ const StyledPostText = styled(PostText)`
 `;
 
 const Separator = styled.hr`
-    && {
+    &&& {
```
Comment (lieut-data) [inline webapp/src/components/update_post.tsx:120]: More specificity to overcome: 
image

File: webapp/src/components/update_post.tsx [hunk 2]
```diff
@@ -117,10 +117,11 @@ const StyledPostText = styled(PostText)`
...
         border: none;
         height: 1px;
         background: rgba(var(--center-channel-color-rgb), 0.61);
+        margin: 0;
```
Comment (lieut-data) [inline webapp/src/components/update_post.tsx:124]: Override new Bootstrap scaffolding:
image

File: webapp/src/components/update_post.tsx [hunk 3]
```diff
@@ -117,10 +117,11 @@ const StyledPostText = styled(PostText)`
 `;
 
 const Separator = styled.hr`
-    && {
+    &&& {
         border: none;
         height: 1px;
         background: rgba(var(--center-channel-color-rgb), 0.61);
```
Comment (calebroseland) [inline webapp/src/components/update_post.tsx:123]: Seemed a little bolder than the usual divider, probably typo during feature—should be 0.16 per [designs](https://www.figma.com/file/OJeUZDfESLlXgblbn7Beac/Better-Status-Updates?node-id=1027%3A333472&t=2OvhTlqVtl7MFdOs-4).

```suggestion
        background: rgba(var(--center-channel-color-rgb), 0.16);
```

**Review by calebroseland — APPROVED** [review]
LGTM! Though it would be nice to fix the bg color while we're here.

File: webapp/src/components/update_post.tsx (hunk 2 above)
Comment (calebroseland) [inline webapp/src/components/update_post.tsx:124]: Though maybe this, too:

```suggestion
        margin: 12px 0;
```

CleanShot 2022-11-14 at 10 00 27@2x
CleanShot 2022-11-14 at 10 00 22@2x

**Review by abhijit-singh — CHANGES_REQUESTED** [review]
There seems to be a 20% opacity applied on top of the `background: rgba(var(--center-channel-color-rgb), 0.16);`, which is making the line very light. 

I think we can either add an opacity:1 attribute to this instance or change the color to  `background: rgba(var(--center-channel-color-rgb), 0.8);`.
image

**Comment by lieut-data** [comment]
Wow, good eye, @abhijit-singh! Now looks like:

image

**Review by abhijit-singh — APPROVED** [review]
💯

**Pull request merged**